packages/eslint-plugin-js/rules/linebreak-style/linebreak-style.ts

```diff
@@ -3,14 +3,15 @@
  * @author Erik Mueller
  */
 
+import type { TSESLint, TSESTree } from '@typescript-eslint/utils'
 import { createGlobalLinebreakMatcher } from '../../utils/ast-utils'
+import { createRule } from '../../utils/createRule'
 
 // ------------------------------------------------------------------------------
 // Rule Definition
 // ------------------------------------------------------------------------------
 
-/** @type {import('eslint').Rule.RuleModule} */
-export default {
+export default createRule({
```

Comment:
```suggestion
export default createRule<MessageIds, RuleOptions>({
```

It would be better to add these.